students/Steven/lesson08/circle_class.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+"""
+The goal is to create a class that represents a simple circle.
+
+A Circle can be defined by either specifying the radius or the diameter, and the user can query the circle for either its radius or diameter.
+
+Other abilities of a Circle instance:
+
+Compute the circle’s area.
+Print the circle and get something nice.
+Be able to add two circles together.
+Be able to compare two circles to see which is bigger.
+Be able to compare to see if they are are equal.
+(follows from above) be able to put them in a list and sort them.
+"""
+
+import math
+
+class Circle():
+    # Circle object with a given radius
+    def __init__(self, radius):
+        self.radius = radius
+
+    @property
+    def get_diameter(self):
+        return self.radius * 2
+
+    @get_diameter.setter
+    def get_diameter(self, value):
```

Comment:
since its a property you want to use nouns for the name (for functions you'd use verbs like get_*)